chain/client/src/chunks_delay_tracker.rs

```diff
@@ -19,174 +18,76 @@ pub(crate) struct ChunksDelayTracker {
     heights: BTreeMap<BlockHeight, HeightInfo>,
```

Comment:
you might want to update the comment above

Reply:
also - maybe rather than 'heights' I'd call it 'blocks_in_progress'?

And then 'remove_old_entries' would be something like 'remove_completed_blocks'.